clang/test/SemaSYCL/sycl-dllimport-dllexport.cpp

```diff
@@ -40,10 +40,10 @@ int zoo() __attribute__((dllimport));
 int  __declspec(dllexport) foo(int a) {
```

Comment:
Comment before this function seems unnecessary after your changes.

Reply:
Comment removed.